plugins/nf-schema/src/main/nextflow/validation/config/ValidationConfig.groovy

```diff
@@ -20,18 +20,20 @@ class ValidationConfig {
     final public Boolean failUnrecognisedHeaders
     final public String  parametersSchema
     final public Boolean showHiddenParams
+    final public Integer maxValueLength
     final public HelpConfig help
     final public SummaryConfig summary
 
     final public List<String> ignoreParams
 
     ValidationConfig(Map map, Map params){
         def config = map ?: Collections.emptyMap()
-        lenientMode             = config.lenientMode                ?: false
-        monochromeLogs          = config.monochromeLogs             ?: false
-        failUnrecognisedParams  = config.failUnrecognisedParams     ?: false
-        failUnrecognisedHeaders = config.failUnrecognisedHeaders    ?: false
-        showHiddenParams        = config.showHiddenParams           ?: false
+        lenientMode             = config.lenientMode                                    ?: false
+        monochromeLogs          = config.monochromeLogs                                 ?: false
+        failUnrecognisedParams  = config.failUnrecognisedParams                         ?: false
+        failUnrecognisedHeaders = config.failUnrecognisedHeaders                        ?: false
+        showHiddenParams        = config.showHiddenParams                               ?: false
+        maxValueLength          = config.maxValueLength && config.maxValueLength >= 0   ? config.maxValueLength : 150
```

Comment:
does a value of 0 make sense? maybe we should set a minimum to 1

Reply:
Good point!

Reply:
Fixed!